media-libs/noise-suppression-for-voice/noise-suppression-for-voice-1.10-r1.ebuild

```diff
@@ -19,8 +19,11 @@ fi
 LICENSE="GPL-3+"
 SLOT="0"
 
-IUSE="+ladspa lv2 vst vst3 test"
-REQUIRED_USE="|| ( ladspa lv2 vst vst3 )"
+IUSE="+ladspa lv2 vst vst3 +pipewire test"
```

Comment:
is there much point in having ladspa separate? Can't we just always do ladspa but control pipewire with pipewire, or even just drop the pipewire USE and install the file unconditionally to .avail?

Reply:
I added the pipewire flag and dep since I was worried about .avail not being owned by media-video/pipewire if noise-suppression-for-voice is installed after pipewire, but if that's not a concern then the USE flag is not needed. I suppose we could also drop the ladspa flag since it's not too heavy to build compared to the other plugins (doing so would also clean up REQUIRED_USE).